client/post-editor/editor-sharing/accordion.jsx

```diff
@@ -132,10 +141,16 @@ export default connect(
 			true !== getSiteOption( state, siteId, 'publicize_permanently_disabled' ) &&
 			postTypeSupports( state, siteId, postType, 'publicize' )
 		);
+		const isSharingActive = false !== isJetpackModuleActive( state, siteId, 'sharedaddy' );
+		const isLikesActive = false !== isJetpackModuleActive( state, siteId, 'likes' );
+		const isJetpack = false !== isJetpackSite( state, siteId );
 
 		return {
+			connections: getSiteUserConnections( state, siteId, userId ),
+			isSharingAndLikesDisabled: ! isSharingActive && ! isLikesActive,
+			isJetpack,
```

Comment:
Strange I thought I had left a comment here.  Sorry, too many tabs.  What I had written was I personally think the approach here is quite legible at a quick glance, but will change it up because I'm fine either way.